pegasus/test/fixtures/fake_dashboard.rb

```diff
@@ -247,6 +247,13 @@ def create_table(name, options)
   end
   ActiveRecord::ConnectionAdapters::Mysql2Adapter.prepend SchemaTableFilter
 
+  # Patch Mysql2Adapter to stub create_view when loading the schema.
```

Comment:
@wjordan Is this an acceptable mechanism for stubbing the Scenic `create_view` method?